h/presenters/annotation_json.py

```diff
@@ -9,68 +9,59 @@
 
 
 class AnnotationJSONPresenter(AnnotationBasePresenter):
-
     """Present an annotation in the JSON format returned by API requests."""
 
     def __init__(self, annotation_context, formatters=None):
         super().__init__(annotation_context)
 
-        self._formatters = []
-
-        if formatters is not None:
-            self._formatters.extend(formatters)
+        self._formatters = tuple(formatters or [])
```

Comment:
I think this is all intended to make this immutable, which I think a tuple does quite well

Reply:
The code wasn't going out of its way, as far as I can see, to make `self._formatters` immutable was it? It just happens not to mutate it / has no intention of ever mutating it?

Anyway, if it's not mutated and not intended to be mutable then a tuple seems more correct than a list since it signals to the reader that this is not going to be mutated. I think that's good usage of different builtin types

Reply:
Yeah it was. There was a test dedicated to this. The original PR that brought it in also mentioned it as an explicit goal?

Honestly, I don't think it matters at all, and we can just do `self.formatter = formatters` like you normally would, but this doesn't seem like much.